eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -334,10 +304,49 @@ jobs:
               continueOnError: true
               condition: and(succeeded(), eq(variables['VerifyChangeLogForRelease'], 'true'))
 
+        - pwsh: |
+            $packageJsonDest = Join-Path -Path $(Build.ArtifactStagingDirectory) PackageInfo
+            $packageJsonSource = Join-Path -Path $(Build.ArtifactStagingDirectory) packages/PackageInfo
+            mkdir $packageJsonDest
+            Copy-Item $packageJsonSource/* $packageJsonDest -Force
+          displayName: Copy Package Json
         - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
           parameters:
             ArtifactPath: $(Build.ArtifactStagingDirectory)/packages
             ArtifactName: packages
+        - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
+          parameters:
+            ArtifactPath: $(Build.ArtifactStagingDirectory)/PackageInfo
+            ArtifactName: PackageInfo
+        
+        - task: Powershell@2
+          inputs:
+            filePath: $(System.DefaultWorkingDirectory)/eng/common/scripts/Create-APIReview.ps1
+            arguments: >
+              -ArtifactList ('${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json | Select-Object Name)
+              -OutPath $(Build.ArtifactStagingDirectory)
+              -ApiKey "$(azuresdk-apiview-apikey)"
+              -SourceBranch $(Build.SourceBranchName)
+              -DefaultBranch $(DefaultBranch)
+              -ParserPath $(System.DefaultWorkingDirectory)/parser/RelWithDebInfo/ParseAzureSdkCpp.exe
+              -ServicePath $(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}
+              -BuildId $(Build.BuildId)
+              -RepoName "$(Build.Repository.Name)"
+            pwsh: true
+            workingDirectory: $(Pipeline.Workspace)
+          displayName: Create API Review
+          condition: >-
+            and(
+              succeeded(),
+              ne(variables['Skip.CreateApiReview'], 'true'),
+              ne(variables['Build.Reason'],'PullRequest'),
+              eq(variables['System.TeamProject'], 'internal')
+            )
+        - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+          - ${{ each artifact in parameters.Artifacts }}:
+            - template: /eng/common/pipelines/templates/steps/detect-api-changes.yml
+              parameters:
+                ArtifactPath: $(Build.ArtifactStagingDirectory)\packages
```

Comment:
```suggestion
                ArtifactPath: $(Build.ArtifactStagingDirectory)/packages
```

Reply:
this one doesn't work because in the detect-api-changes, there's a place where it uses -replace artifactPath, and since we're using windows os, we'll need the backwards slash